grizzly/replay/replay.py

```diff
@@ -310,6 +312,21 @@ def run(
             )
             server_map.set_redirect("grz_start", "grz_harness", required=False)
 
+        if services:
+
+            def create_cb(url):
+                """Creates lambda return URL for use in loops"""
+                return lambda _: url
+
+            for service in services:
+                if isinstance(service, WebTransportServer):
+                    server_map.set_dynamic_response(
+                        "grz_webtransport_server",
+                        create_cb(b"https://127.0.0.1:%d" % (service.port,)),
```

Comment:
I think one of Tyson's comments got eaten regarding the lambdas.... This is wrong. Either create the function to return a bytestring and pass that, or use a lambda and ignore the pylint warning.

```python

def create_url():
    return f"https://127.0.0.1:{service.port}".encode("ascii")

server_map.set_dynamic_response("grz_webtransport_server", create_url, ...)
```